robocop/exceptions.py

```diff
@@ -27,9 +27,9 @@ def __init__(self, source):
         super().__init__(msg)
 
 
-class NestedArgumentFileError(RobocopFatalError):
+class CircularArgumentFileError(RobocopFatalError):
```

Comment:
These changes are in the scope of another PR #774 and should be removed from here.